### PR TITLE
Update links to "Community Applications" in README to point to "examples/stable" #409

### PR DESCRIPTION
**Pull request by rahulsidgondapatil**
#409   Change Overview
Updated links to existing examples and added new examples.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [x] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #409 

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,11 +30,15 @@ your storage infrastructure.
...
-- **[Elasticsearch](./examples/helm/kanister/kanister-elasticsearch)**
-- **[MongoDB](./examples/helm/kanister/kanister-mongodb-replicaset)**
-- **[MySQL](./examples/helm/kanister/kanister-mysql)**
-- **[PostgreSQL](./examples/helm/kanister/kanister-postgresql)**
+- **[Cassandra](./examples/stable/cassandra)**
+- **[Couchbase](./examples/stable/couchbase)**
+- **[Elasticsearch](./examples/stable/elasticsearch)**
+- **[FoundationDB](./examples/stable/foundationdb)**
+- **[MongoDB](./examples/stable/mongodb)**
+- **[MySQL-deploymentconfig](./examples/stable/mysql-deploymentconfig)**
```
Comment (viveksinghggits) [inline README.md]: Hey @rahulsidgondapatil 
Let's change `MySQL-deploymentconfig` to `MySQL on OpenShift using DeploymentConfig`

File: README.md [hunk 2]
```diff
@@ -30,11 +30,15 @@ your storage infrastructure.
...
+- **[MySQL](./examples/stable/mysql)**
+- **[PostgreSQL-wale](./examples/stable/postgresql-wale)**
```
Comment (PrasadG193) [inline README.md]: `PostgreSQL-wale` =>  `PostgreSQL with Point In Time Recovery (PITR)`

Comment (rahulsidgondapatil) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (rahulsidgondapatil) [reply README.md]: done

**Pull request merged**